src/main/java/de/dhbw/karlsruhe/services/ScannerService.java

```diff
@@ -0,0 +1,34 @@
+package de.dhbw.karlsruhe.services;
+
+import java.util.Scanner;
+
+public class ScannerService {
+    private static Scanner scanner ;
+
+    public ScannerService(){
+        if (scanner == null) {
+            scanner = new Scanner(System.in);
+        }
+    }
+
+    public Scanner getScanner() {
+        return this.scanner;
+    }
+
+    public String getLine() {
+        String input = this.scanner.nextLine();
+        return input;
+    }
+
+    public int getInt() {
+        int input =this.scanner.nextInt();
```

Comment:
s.o. Methode zu nextInt() umbenennen. 

Ich versteh noch nicht ganz wofür  der Aufruf this.getLIne() benötigt wird. Ist das nicht überflüssig?
 
Beide Punkte gelten für die restlichen Methoden der Klasse auch ;) 
Sonst passt es so für mich :)

Reply:
Beim einlesen von Int Werten über scanner.nextInt() wird der Zeilenumbruch nicht mitgelesen. Es bleibt somit "in der Eingabe". Wird nun als nächstes ein nextLine() aufgerufen, wird der Zeilenumbruch gelesen und der Benutzer kann keine richtige Eingabe machen.

Die Methoden sind aber jetzt aber eh überflüssig, da wir die Methoden des statischen scanner Objekts benutzen